src/science/soilsnow/cbl_surfbv.F90

```diff
@@ -10,6 +10,8 @@ SUBROUTINE surfbv (dels, met, ssnow, soil, veg, canopy )
 
 USE smoisturev_mod,               ONLY: smoisturev
     USE cable_common_module
+    USE grid_constants_mod_cbl, ONLY : lakes_cable
```

Comment:
My only comment with this is that we've just pivoted in AM3 to using a namelist input to define lakes_cable. Something which will need to be ported to offline anyway - so the is a fair solution until that happens